src/core/shape/vertex.js

```diff
@@ -314,17 +314,29 @@ p5.prototype.beginShape = function(kind) {
  * crescent-shaped line in middle of canvas. Points facing left.
  * white crescent shape in middle of canvas. Points facing left.
  */
+
+/**
+ * @method bezierVertex
+ * @param  {Number} x2
+ * @param  {Number} y2
+ * @param  {Number} x3
+ * @param  {Number} y3
+ * @param  {Number} x4
+ * @param  {Number} y4
+ * @chainable
+ */
```

Comment:
is this signature the same as above in the main yuidoc entry? it can be omitted completely then. only the alternatives are listed below.